Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error while parsing easi model file.
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error while parsing easi model file.
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error while parsing easi model file.
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error while parsing easi model file.
terminate called after throwing an instance of 'std::runtime_error'
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
  what():  Error while parsing easi model file.
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error while parsing easi model file.
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error while parsing easi model file.
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error while parsing easi model file.
terminate called after throwing an instance of 'std::runtime_error'
terminate called after throwing an instance of 'std::runtime_error'
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
  what():  Error while parsing easi model file.
  what():  Error while parsing easi model file.
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error while parsing easi model file.
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
terminate called after throwing an instance of 'std::runtime_error'
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error while parsing easi model file.
  what():  Error while parsing easi model file.
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error while parsing easi model file.
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error while parsing easi model file.
terminate called after throwing an instance of 'std::runtime_error'
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
  what():  Error while parsing easi model file.
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error while parsing easi model file.
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error while parsing easi model file.
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error while parsing easi model file.
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error while parsing easi model file.
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error while parsing easi model file.
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
Haiti_fault.yaml: yaml-cpp: error at line 103, column 11: illegal map value
terminate called after throwing an instance of 'std::runtime_error'
terminate called after throwing an instance of 'std::runtime_error'
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error while parsing easi model file.
  what():  Error while parsing easi model file.
  what():  Error while parsing easi model file.
srun: error: i04r03c05s03: task 18: Aborted (core dumped)
srun: launch/slurm: _step_signal: Terminating StepId=3984667.0
srun: error: i04r03c05s07: task 22: Aborted (core dumped)
srun: error: i04r03c05s02: task 17: Aborted (core dumped)
srun: error: i04r03c05s01: task 16: Aborted (core dumped)
srun: error: i04r03c03s02: task 8: Aborted (core dumped)
srun: error: i04r03c02s08: task 2: Aborted (core dumped)
srun: error: i04r03c05s05: task 20: Aborted (core dumped)
srun: error: i04r03c03s01: task 7: Aborted (core dumped)
srun: error: i04r03c03s03: task 9: Aborted (core dumped)
srun: error: i04r03c03s04: task 10: Aborted (core dumped)
srun: error: i04r03c02s07: task 1: Aborted (core dumped)
srun: error: i04r03c02s09: task 3: Aborted (core dumped)
srun: error: i04r03c03s05: task 11: Aborted (core dumped)
srun: error: i04r03c05s06: task 21: Aborted (core dumped)
srun: error: i04r03c02s12: task 6: Aborted (core dumped)
srun: error: i04r03c02s11: task 5: Aborted (core dumped)
srun: error: i04r03c04s10: task 13: Aborted (core dumped)
srun: error: i04r03c02s06: task 0: Aborted (core dumped)
srun: error: i04r03c05s04: task 19: Aborted (core dumped)
srun: error: i04r03c05s08: task 23: Aborted (core dumped)
srun: error: i04r03c04s11: task 14: Aborted (core dumped)
srun: error: i04r03c04s12: task 15: Aborted (core dumped)
srun: error: i04r03c03s06: task 12: Aborted (core dumped)
srun: error: i04r03c02s10: task 4: Aborted (core dumped)
cp: cannot create directory '/hppfs/scratch/01/di35poq/haiti-rupture-outputs/dynamic-rupture-outputs/variable_SH_max/jobid_3984667/logs': No such file or directory
/var/spool/slurm/job3984667/slurm_script: line 80: pushd: /hppfs/scratch/01/di35poq/haiti-rupture-outputs/dynamic-rupture-outputs/variable_SH_max/jobid_3984667: No such file or directory
Traceback (most recent call last):
  File "/dss/dsshome1/01/di35poq/.local/bin/seissol_output_extractor", line 8, in <module>
    sys.exit(main())
  File "/dss/dsshome1/01/di35poq/.local/lib/python3.8/site-packages/seissolxdmfwriter/seissol_output_extractor.py", line 143, in main
    sx = SeissolxdmfExtended(args.xdmfFilename)
  File "/dss/dsshome1/01/di35poq/.local/lib/python3.8/site-packages/seissolxdmf/seissolxdmf.py", line 14, in __init__
    with open (xdmfFilename, "r") as fid:
FileNotFoundError: [Errno 2] No such file or directory: 'output-surface.xdmf'
Traceback (most recent call last):
  File "/dss/dsshome1/01/di35poq/soft/calc-moment-rate_supermuc.py", line 30, in find_xdmffile_wahweap
    xdmfFile = glob.glob(datadir + '*/jobid_'+jobid+'*/*extracted-fault.xdmf')[0]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/dss/dsshome1/01/di35poq/soft/calc-moment-rate_supermuc.py", line 211, in <module>
    generateMoment(jobid_list)
  File "/dss/dsshome1/01/di35poq/soft/calc-moment-rate_supermuc.py", line 87, in generateMoment
    xdmfFilename, momentCsvFile = find_xdmffile_wahweap(jobid)
  File "/dss/dsshome1/01/di35poq/soft/calc-moment-rate_supermuc.py", line 34, in find_xdmffile_wahweap
    xdmfFile = glob.glob(datadir + '*/jobid_'+jobid+'*/output-fault.xdmf')[0]
IndexError: list index out of range
Traceback (most recent call last):
  File "/dss/dsshome1/01/di35poq/soft/calc-R-rake_supermuc.py", line 83, in <module>
    jobdir, xdmfFile, momentCsvFile = find_xdmffile_wahweap(jobid)
  File "/dss/dsshome1/01/di35poq/soft/calc-R-rake_supermuc.py", line 32, in find_xdmffile_wahweap
    xdmfFile = glob.glob(datadir + '*/jobid_'+jobid+'*/output-fault.xdmf')[0]
IndexError: list index out of range
[0m[31m(   3.992s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   3.992s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x35fb590)
  Debug: Off
  Modified Time: 92784
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   3.992s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   3.992s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35ef930): Algorithm vtkXdmfReader (0x35eccd0) returned failure for request: vtkInformation (0x35ee7e0)
  Debug: Off
  Modified Time: 92671
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   3.992s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   3.992s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35ef930): Algorithm vtkXdmfReader (0x35eccd0) returned failure for request: vtkInformation (0x35ee7e0)
  Debug: Off
  Modified Time: 92671
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   3.992s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   3.992s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x35fb590)
  Debug: Off
  Modified Time: 92784
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   3.992s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   3.992s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35ef930): Algorithm vtkXdmfReader (0x35eccd0) returned failure for request: vtkInformation (0x35ee7e0)
  Debug: Off
  Modified Time: 92671
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   3.992s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   3.992s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35ef930): Algorithm vtkXdmfReader (0x35eccd0) returned failure for request: vtkInformation (0x35ee7e0)
  Debug: Off
  Modified Time: 92671
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   3.994s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   3.994s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x35fb590)
  Debug: Off
  Modified Time: 92784
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   3.994s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   3.994s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35ef930): Algorithm vtkXdmfReader (0x35eccd0) returned failure for request: vtkInformation (0x35ee7e0)
  Debug: Off
  Modified Time: 92671
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   3.994s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   3.994s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35ef930): Algorithm vtkXdmfReader (0x35eccd0) returned failure for request: vtkInformation (0x35ee7e0)
  Debug: Off
  Modified Time: 92671
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   4.634s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   4.634s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x4f0b8d0)
  Debug: Off
  Modified Time: 154810
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   4.634s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   4.634s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x4f0b8d0)
  Debug: Off
  Modified Time: 154810
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[33m(   4.822s) [paraview        ]vtkSMColorMapEditorHelp:772   WARN| Failed to determine the LookupTable being used.[0m
[0m[31m(   4.823s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   4.823s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x5a8a0e0)
  Debug: Off
  Modified Time: 878740
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   4.823s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkCompositeDataPipeline (0x59f70a0): Algorithm vtkPVGeometryFilter (0x5b55810) returned failure for request: vtkInformation (0x59f7f60)
  Debug: Off
  Modified Time: 878744
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   4.823s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   4.823s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x59ad920)
  Debug: Off
  Modified Time: 878808
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   4.823s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkCompositeDataPipeline (0x59caba0): Algorithm vtkPVGeometryFilter (0x599b510) returned failure for request: vtkInformation (0x59cbc70)
  Debug: Off
  Modified Time: 878813
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   4.823s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   4.823s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x5420760)
  Debug: Off
  Modified Time: 878871
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   4.831s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkCompositeDataPipeline (0x59f70a0): Algorithm vtkPVGeometryFilter (0x5b55810) returned failure for request: vtkInformation (0x59f7f60)
  Debug: Off
  Modified Time: 878744
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   4.831s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkCompositeDataPipeline (0x59caba0): Algorithm vtkPVGeometryFilter (0x599b510) returned failure for request: vtkInformation (0x59cbc70)
  Debug: Off
  Modified Time: 878813
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[33m(   4.831s) [paraview        ]vtkSMColorMapEditorHelp:772   WARN| Failed to determine the LookupTable being used.[0m
[0m[31m(   4.831s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkCompositeDataPipeline (0x59f70a0): Algorithm vtkPVGeometryFilter (0x5b55810) returned failure for request: vtkInformation (0x59f7f60)
  Debug: Off
  Modified Time: 878744
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   4.832s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkCompositeDataPipeline (0x59caba0): Algorithm vtkPVGeometryFilter (0x599b510) returned failure for request: vtkInformation (0x59cbc70)
  Debug: Off
  Modified Time: 878813
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   4.837s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   4.837s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x63378d0)
  Debug: Off
  Modified Time: 882186
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   4.838s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   4.838s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x63613e0)
  Debug: Off
  Modified Time: 882368
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   4.838s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   4.838s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x63613e0)
  Debug: Off
  Modified Time: 882368
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   4.953s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   4.953s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x63613e0)
  Debug: Off
  Modified Time: 882368
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   4.953s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   4.953s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x63613e0)
  Debug: Off
  Modified Time: 882368
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[33m(   4.964s) [paraview        ]vtkSMColorMapEditorHelp:772   WARN| Failed to determine the LookupTable being used.[0m
[0m[33m(   4.970s) [paraview        ]vtkSMColorMapEditorHelp:170   WARN| Could not determine array range.[0m
[0m[33m(   4.970s) [paraview        ]vtkSMColorMapEditorHelp:170   WARN| Could not determine array range.[0m
[0m[31m(   4.981s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   4.981s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x71ebee0)
  Debug: Off
  Modified Time: 1029696
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   4.982s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   4.982s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x71ed5a0)
  Debug: Off
  Modified Time: 1029878
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   4.982s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   4.982s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x71ed5a0)
  Debug: Off
  Modified Time: 1029878
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   5.094s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   5.094s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x71ed5a0)
  Debug: Off
  Modified Time: 1029878
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   5.094s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   5.094s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x71ed5a0)
  Debug: Off
  Modified Time: 1029878
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   5.104s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   5.104s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x71ed5a0)
  Debug: Off
  Modified Time: 1029878
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   5.104s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   5.104s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x71ed5a0)
  Debug: Off
  Modified Time: 1029878
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   5.108s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   5.108s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x6dccec0)
  Debug: Off
  Modified Time: 1171896
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
[0m[31m(   5.108s) [paraview        ]      vtkXdmfReader.cxx:253    ERR| vtkXdmfReader (0x35eccd0): Error opening file output_jobid_3984667_extracted-fault.xdmf[0m
[0m[31m(   5.108s) [paraview        ]       vtkExecutive.cxx:729    ERR| vtkPVCompositeDataPipeline (0x35fbfa0): Algorithm vtkFileSeriesReader (0x35f9a10) returned failure for request: vtkInformation (0x6dccec0)
  Debug: Off
  Modified Time: 1171896
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0

[0m
Traceback (most recent call last):
  File "/dss/dsshome1/01/di35poq/soft/plot_initial_stress_profile.py", line 58, in <module>
    data = pd.read_csv(csvfile)
  File "/dss/dsshome1/01/di35poq/.local/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 912, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/dss/dsshome1/01/di35poq/.local/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 577, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/dss/dsshome1/01/di35poq/.local/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1407, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/dss/dsshome1/01/di35poq/.local/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1679, in _make_engine
    return mapping[engine](f, **self.options)
  File "/dss/dsshome1/01/di35poq/.local/lib/python3.8/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 93, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "pandas/_libs/parsers.pyx", line 557, in pandas._libs.parsers.TextReader.__cinit__
pandas.errors.EmptyDataError: No columns to parse from file
/var/spool/slurm/job3984667/slurm_script: line 95: popd: directory stack empty
cp: target '/hppfs/scratch/01/di35poq/haiti-rupture-outputs/dynamic-rupture-outputs/variable_SH_max/jobid_3984667/logs/' is not a directory
